Formula/mongodb-community@5.0.rb

```diff
@@ -1,14 +1,16 @@
-class MongodbCommunityAT34 < Formula
+class MongodbCommunityAT50 < Formula
```

Comment:
I don't think there should be a `Formula/mongodb-community@5.0.rb` if we are following the pattern from the 4.4 GA.  If we are doing that, Instead you need to make a symlink from `Aliases/mongodb-community@5.0.rb` to `Formula/mongodb-community@5.0.rb` and remove `Aliases/mongodb-community@4.4.rb`. I think something also needs to be done with `Aliases/mongodb-mongocryptd@4.4.rb`. Yes, it's weird.

Reply:
Our documentation says to install the `@5.0` version specifically, which is why I made it. There's a bit of nuance surrounding the class name and the target, so I don't think a symlink would work in this instance. I think it's better to just update both the non-targeted and targeted files going forward.

Reply:
Oh, yeah, I missed `mongocryptd` completely. I'll get that.

Reply:
Right, but the way it (appears to) work is that you want `mongodb-community` with no adornment to be the thing that installs 5.0. Then you make an Alias entry for that called @5.0. Have a look at how it is structured currently w.r.t. 4.4.

Reply:
Ditto re `Aliases/mongodb-mongocryptd@4.4` to `Aliases/mongodb-mongocryptd@5.0`

Reply:
Ah, right. I had forgotten how that worked for some reason. I think Daniel's last commit that we reverted added a new class rather than update the Aliases, which is why I got confused.